backend/test/scans.test.ts

```diff
@@ -93,6 +93,13 @@ describe('scan', () => {
         isGranular: true,
         isSingleScan: false
       }).save();
+      const scan3 = await Scan.create({
```

Comment:
Can you instead make a new test case called "`it('list by regular user should exclude single scans')`" and put this test in that test case? It's best to keep each test case isolated.